tests/encapsed/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -23,6 +23,6 @@ $encapsHereDoc2 = <<<STR
 Hello foo $a bar
 new line
 STR;
-$encapsedOffset = $a->bar$b;
+$encapsedOffset = $a->{bar$b};
```

Comment:
Is `$a->bar$b` valid PHP?

Reply:
I don't think so, it seems to be a parse error in PHP7

```
php > $a = new stdClass;
php > $a->foobar = 1;
php > $b = 'bar';
php > echo $a->{"foo$b"};
1
php > echo $a->foo$b;

Parse error: syntax error, unexpected '$b' (T_VARIABLE), expecting ',' or ';' in php shell code on line 1
php > echo $a->{foo$b};

Parse error: syntax error, unexpected '$b' (T_VARIABLE) in php shell code on line 1
```

Reply:
:+1: i'll remove it from the tests